src/bindings/python/flux/util.py

```diff
@@ -663,14 +663,18 @@ def __init__(self, name, initial_dict=None):
         if initial_dict:
             self.dict = dict(initial_dict)
 
-        #  Build config search path in order of reverse precedence
-        #  based on XDG specification:
-        self.searchpath = (
-            ["/etc/xdg/flux"]
-            + get_searchpath("XDG_CONFIG_DIRS", "/flux")
-            + [f"{Path.home()}/.config/flux"]
-            + get_searchpath("XDG_CONFIG_HOME", "/flux")
-        )
```

Comment:
with the fixup i don't think you need `get_searchpath()` anymore

Reply:
Yeah, sorry, still a work in progress!